Boundaries.geojson

```diff
@@ -441,8 +441,7 @@
-  { "type": "Feature", "properties": { "id": "UHMM", "oceanic": "0", "label_lon": "166.0", "label_lat": "65.6", "region": "EMEA", "division": "VATRUS" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ 164.0, 62.07 ], [ 162.583333, 61.583333 ], [ 160.166667, 60.583333 ], [ 157.0, 59.333333 ], [ 156.0, 58.916667 ], [ 153.983333, 58.016667 ], [ 150.0, 56.0 ], [ 150.0, 54.873333 ], [ 146.0, 56.0 ], [ 143.416667, 56.416667 ], [ 145.75, 58.0 ], [ 146.0, 61.0 ], [ 145.0, 62.0 ], [ 145.6, 63.833333 ], [ 149.666667, 64.0 ], [ 158.0, 65.75 ], [ 158.0, 66.55 ], [ 162.0, 66.55 ], [ 165.0, 68.666667 ], [ 168.0, 70.5 ], [ 168.0, 72.5 ], [ 158.0, 72.0 ], [ 143.583333, 73.2 ], [ 132.0, 74.333333 ], [ 114.716667, 76.333333 ], [ 114.75, 78.0 ], [ 169.0, 90.0 ], [ 180.0, 80.0 ], [ 180.0, 61.366666 ], [ 177.0, 61.15 ], [ 164.0, 62.07 ] ] ], [ [ [ -168.973056, 65.0 ], [ -175.0, 62.583333 ], [ -177.751666, 61.366666 ], [ -180.0, 61.366666 ], [ -180.0, 80.0 ], [ -168.973333, 72.0 ], [ -168.973056, 65.0 ] ] ] ] } },
-  { "type": "Feature", "properties": { "id": "UHPP", "oceanic": "0", "label_lon": "164.3", "label_lat": "57.0", "region": "EMEA", "division": "VATRUS" }, "geometry": { "type": "MultiPolygon", "coordinates": [ [ [ [ 164.0, 62.07 ], [ 177.0, 61.15 ], [ 180.0, 61.366666 ], [ 180.0, 60.0 ], [ 170.2, 54.816667 ], [ 170.0, 54.678333 ], [ 169.0, 54.0 ], [ 159.0, 50.083333 ], [ 150.0, 45.0 ], [ 150.0, 49.0 ], [ 150.0, 52.166667 ], [ 150.0, 56.0 ], [ 153.983333, 58.016667 ], [ 156.0, 58.916667 ], [ 157.0, 59.333333 ], [ 160.166667, 60.583333 ], [ 162.583333, 61.583333 ], [ 164.0, 62.07 ] ] ], [ [ [ -180.0, 61.366666 ], [ -177.751666, 61.366666 ], [ -180.0, 60.0 ], [ -180.0, 61.366666 ] ] ] ] } },
```

Comment:
Removoal of a FIR also requires the FIR to be removed in VATSpy.dat.
You also have to assosiate all the airports that has UHPP as a FIR to another FIR.

Reply:
Ok, I see the changes is done in #608 
But please next time do not seperate changes into two pull requests.